app/utils/newsFeed/redis/mappings.ts

```diff
@@ -298,3 +305,24 @@ export const mapToRedisUser = async (userId: string): Promise<RedisUser> => {
   const user = await getInnoUserByProviderId(userId);
   return user;
 };
+
+const updateImageUrl = (imageUrl: string | undefined): string | undefined => {
+  if (imageUrl && imageUrl.startsWith('http://127.0.0.1:1337/')) {
+    return imageUrl.replace('http://127.0.0.1:1337', '');
+  }
+  return imageUrl;
+};
+
+const updateImageUrls = (item: any) => {
```

Comment:
We should avoid `any` as it disables type checking and can cause errors while the app is running. 

Here you are replacing either image urls for images of type `ImageFormats` or the profile pictures of users. So try to create correctly typed methods for this:
```
const mapImageToRelativeUrls = (image: ImageFormats) => {};
const mapUserImageToRelativeUrls = (user: RedisUser) => {};
```

these can than be used instead of the `updateImageUrls` method.